### DISCUSSION TITLE
ability to track medication history

### DISCUSSION THREAD

**Discussion by caj411** [diegopvlk/Dosage#53] [Ideas]
I like to track what I have taken because as my body gets acclimatized to some medications my doctor will change medications for the same health problem.  I'm not sure how many other people do this, but it would be a nice feature.  As you can see, I have created my own work around by simply using the Name field.  I add the 'X' so they will fall to the bottom, but with a history they would cease to show in my current medication listing.  Again, just an idea for additional feature.
![image](https://github.com/diegopvlk/Dosage/assets/34217875/fdc3a4e5-bbaf-417c-ba21-a91a6092d830)

**Comment by diegopvlk** [comment]
Do you want them to stop showing on today tab? If that's the case, set an end date in duration that comes before today.

**Comment by caj411** [comment]
That works fine, didn't think about that.  I assumed that once the duration ended they would disappear, but I didn't realize they would stay active in the treatment section.  Thanks, I'm learning.
